config/jobs/kubernetes/sig-network/ci-e2e-gce-netd.yaml

```diff
@@ -0,0 +1,231 @@
+presets:
```

Comment:
you need to define a label for your preset, and apply it in your jobs :-)

Reply:
updated, thanks.